lib/wrap-queries.js

```diff
@@ -19,8 +29,10 @@ function wrap() {
 
       // e.g. print.css
       if (name === 'print') {
+        hasAtRule(src);
         contents = new Buffer('\n@media print {\n' + src + '}');
       } else if (!isNaN(startingNumber)) {
+        hasAtRule(src);
```

Comment:
Are these assertions?  Who are we expecting to catch these?

Maybe we should just warn, and then eat the bad rules?